static/js/apps/nl_interface/query_result.tsx

```diff
@@ -167,6 +176,38 @@ export const QueryResult = memo(function QueryResult(
               pageConfig={chartsData ? chartsData.config : null}
             ></DebugInfo>
           )}
+          {chartsData && chartsData.placeFallback && (
+            <div className="nl-query-info">
+              Sorry, there was no relevant data for &quot;
+              {chartsData.placeFallback.origStr}&quot;. &nbsp; Here is data
+              about &quot;{chartsData.placeFallback.newStr}&quot; instead.
+            </div>
+          )}
+          {chartsData && chartsData.placeSource === "PAST_QUERY" && (
+            <div className="nl-query-info">
+              Could not recognize any place in this query, so using{" "}
+              {chartsData.place.name} from a prior query.
```

Comment:
nit: should this refer to the user's current session? i.e. "... from a prior query in this session"